src/Login/atoms.jsx

```diff
@@ -0,0 +1,14 @@
+import { atom, selector } from 'recoil';
+
```

Comment:
just to remmeber:
make differnet files for different type of atoms to easy navigation in them